src/oc_bifrost/apps/bifrost/src/bifrost_sup.erl

```diff
@@ -19,12 +19,16 @@ init([]) ->
 
     {ok, Ip} = application:get_env(bifrost, ip),
     {ok, Port} = application:get_env(bifrost, port),
+    Ssl = application:get_env(bifrost, ssl, false),
```

Comment:
It might be worth using envy here (we use it elsewhere, but somehow this never got converted over). It will provide a little bit of input validation.
For example  see. apps/bifrost/src/bifrost_wm_base.erl:127